README.md

```diff
@@ -76,59 +88,64 @@ Here are the steps needed to run Terraform along with examples of each:
 
 `terraform apply -var-file="secret.tfvars"`
 
-Each of these commands includes the `secret.tfvars` containing the sensitive variables needed to connect to the different resources as described in the previous section.
+Each of these commands includes the `secret.tfvars` containing the sensitive variables that are needed to connect to the different resources as described in the previous section.
 
 ## Results
 
 ### Virtual Machines
 
-You will see 5 virtual machines created with static IP addresses in vSphere.
+You see five virtual machines that are created with static IP addresses in vSphere.
 
-![Virtual Machines](images/vsphere-virtual-machines.png)
+<img src="images/vsphere-virtual-machines.png" alt="Virtual Machines">
 
 ### Apache Web Servers
 
 Each Apache server has a custom `index.html` file that includes the hostname of the machine.
-
-![Apache Server Result](images/apache-server-result.png)
+<img src="images/apache-server-result.png" alt="Apache Server Result">
 
 ### AppDynamics Controller
 
-The 5 virtual machines also appear in the AppDynamics controller, each running an Apache Web Server, and all 5 appearing in the AppDynamics controller.
+The five virtual machines appear in the AppDynamics controller, each running an Apache Web Server, and all five appearing in the AppDynamics controller.
 
+<img src="images/appd-machine-agents.png" alt="List of Machine Agents">
 
+Click any of the check box available just before the `OS` column; then click `View Details` to see that the data reported by the Machine Agent to the AppDynamics Controller. 
 
-![List of Machine Agents](images/appd-machine-agents.png)
+You can see the data that is reported by the Machine Agent on `apache-webserver-1`.
 
-Click the check box next to any of the servers and then click `View Details` to see the data reported by the Machine Agent to the AppDynamics Controller. In the example below, we see data reported by the Machine Agent on `apache-webserver-1`.
+<img src="images/appd-web-server-1.png" alt="Data Reported by the Machine Agent">
```

Comment:
alt text edit "Data reported by the AppDynamics Machine Agent including Load Average, CPU, Availability, and Memory"

Reply:
This might be wayyy too many words for alt text, not sure. You can trim as needed.

Reply:
Sure Anne.